lib/executor.rb

```diff
@@ -10,7 +10,7 @@
 
 module Gauge
   class DebugOptions
-    attr_accessor :host, :port, :notify_dispatcher
+    attr_accessor :host, :port, :notify_dispatcher, :socket_path, :skip_wait_for_start
```

Comment:
do we need the `socket_path` accessor? it's not used anywhere

Reply:
We do not need it, but it's used by the debugger gem. And the access of it does not have a safety check. So we need to have this.